### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by alexwilson**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/8wOkpe).

### CODE REVIEW

File: Core/MultiPrice/CurrencyService.php [hunk 1]
```diff
@@ -23,57 +21,56 @@
...
      */
     public function __construct(
         Session $session,
         CurrencyHandler $currencyHandler,
         $defaultUserCurrency,
-        $currencySessionVariableName = "UserPreferredCurrency"
-    )
-    {
+        $currencySessionVariableName = 'UserPreferredCurrency'
+    ) {
```
Comment (joejones6491) [inline Core/MultiPrice/CurrencyService.php:64]: Is that right? I thought the brace was always meant to be on the next line?

**Comment by joejones6491** [comment]
All changes from array() to [] mean that we have to change the php requirements within composer from 5.3 to 5.4!

File: Core/MultiPrice/CurrencyService.php (hunk 1 above)
Comment (alexwilson) [inline Core/MultiPrice/CurrencyService.php:64]: Yes, except for when there are many arguments

**Comment by alexwilson** [comment]
That might not be such a bad thing, as prod runs 5.5 - Is there a particular reason the bundle is set to 5.3?

Happy to re-adjust if it comes to it.

File: Core/MultiPrice/CurrencyService.php (hunk 1 above)
Comment (joejones6491) [inline Core/MultiPrice/CurrencyService.php:64]: Okay that's fine!

**Comment by joejones6491** [comment]
It's okay, I would have thought that it would be better to support a lower version of php if possible?

**Pull request closed without merging**